### PR TITLE
Feature/add signals filter to screener

### PR DESCRIPTION
**Pull request by jjphung**
PR for issue [#13](https://github.com/oscar0812/pyfinviz/issues/13). Adds the signal filter to the `Screener` class and updates test cases in addition to fixing pre-existing failing cases. 

Screenshot 2023-08-04 at 5 34 54 PM
Screenshot 2023-08-04 at 5 35 03 PM

### CODE REVIEW

File: pyfinviz/test/test_quote.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def test_main_AMZN(self):
         self.assertEqual(20, quote.income_statement_df.columns.size)
         self.assertEqual(8, len(quote.income_statement_df))
         self.assertEqual(11, quote.insider_trading_df.columns.size)
-        self.assertEqual(91, len(quote.insider_trading_df))
+        self.assertTrue(not quote.insider_trading_df.empty) # amount of insider trading can change
```
Comment (jjphung) [inline pyfinviz/test/test_quote.py:26]: These tests were already failing, so this was just one alternate solution. Let me know if you have any ideas to account for this ever-changing number.

**Comment by oscar0812** [comment]
@jjphung please view my comments

**Comment by jjphung** [comment]
@oscar0812 Sorry if I am missing something, but I don't see any comments in this PR.
Screenshot 2023-08-07 at 6 55 47 PM

**Comment by oscar0812** [comment]
Check screen.py in your PR. I added a comment.

**Comment by jjphung** [comment]
@oscar0812 I'm still not seeing anything in that file, nor is your comment showing up in the conversation thread here. 🤔 Perhaps you can just repeat that file comment here and reference the line number. Added more screenshots to show no comments:
Screenshot 2023-08-07 at 7 47 55 PM
Screenshot 2023-08-07 at 7 48 01 PM

**Comment by jjphung** [comment]
@oscar0812 Ah I see. I think once you leave a comment you have to submit your review for all your comments and reviews to show up. That's why your status is still "pending review."

1. I added the extra parameter to follow the structure of the URL parameters. `ScreenerFilterOption`s falls under the `&f` URL param while `SignalOption` falls under the `&s` URL. I thought it was nice to make that distinction. 

Example usage:
```
options = [Screener.OptionShortOption.OPTIONABLE, Screener.PriceOption.OVER_USD10]
screener = Screener(filter_options=options, signal_option=Screener.SignalOption.TL_RESISTANCE, view_option=Screener.ViewOption.OVERVIEW)
```
2.  `class SignalOption(ScreenerFilterOption, Enum):` Is the SignalOption class not a child of the ScreenerFilterOption by this definition?

**Review by oscar0812 — APPROVED** [review]
LGTM

**Comment by oscar0812** [comment]
Issue is fixed. Please update to the latest version and confirm:
`pip install pyfinviz==0.13`
I will close this issue once I receive confirmation. Thanks!

**Pull request merged**